Admin/ArticleAdmin.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Admin;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping\ClassMetadataInfo;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\ArticleBundle\Model\Article;
+use Sonata\ArticleBundle\Model\Fragment;
+use Sonata\CoreBundle\Validator\ErrorElement;
+use Sonata\MediaBundle\Admin\ORM\MediaAdmin;
+
+/**
+ * @author Florent Denis <florent.denis@ekino.com>
+ */
+class ArticleAdmin extends AbstractAdmin
```

Comment:
I will extends essentially configureFormFields, showFields, listFields, getTemplate ...
I agree with you, lets set some methods final, and rename this class `AbstractArcticleAdmin` works fine !

The very goal of the whole bundle is to be a starterKit to a much complex article management. This will fit the initial goal.